### ISSUE TITLE
Taste shaming

### ISSUE THREAD

**Issue by Elias-Ainsworth** [pystardust/ani-cli#1396]
**Is your feature request related to a problem? Please describe.**
This is a tool for primarily Linux users and weebs, and it doesn't have a feature that shames you for watching certain anime? For shame!

**Describe the solution you'd like**
Snarky remarks whenever you start watching certain anime, like for the Vinland Saga (which is peak btw) there could be a remark stating "Of course you have no enemies in your mom's basement...except for hygiene." This could be implemented as a notification or as a subtitle on the menu screen (where play next and replay reside).

**Describe alternatives you've considered**
Getting on discord and taste shaming there.

**Additional context**
This is mostly a joke but it would be very funny.

**Comment by GRATHRRAM** [comment]
bruh

**Comment by Elias-Ainsworth** [comment]
lol

**Comment by port19x** [comment]
Honestly, I kinda like the idea ngl.
I'm pretty sure this would amuse @justchokingaround as well.

I'd say we keep to a small set of anime as to not needlessly bloat:
- FLCL ("Testing ani-cli again port19-chan?")
- Lain ("Based")
- Highschool DXD ("Back to the basement gooner")

Your vinland saga can be included as well.

I say this can be done in sub-20 loc, including extensibility or opt-out via env vars

**Comment by Derisis13** [comment]
I also like the idea, but I think there are better places to implement it than ani-cli.
My first hunch would be an mpv script. It can still do the snarky comments either trough the player interface or trough a push notification.

This approach could free the ani-cli staff from having to maintain the feature and it would allow you to collect as many as you wish, even with the help of others.
The only loss would be that references to ani-cli would work not as good, as they would lack context.

TL;DR: I'd prefer this as a separate project that is independent from ani-cli (or at most extends it)

**Comment by port19x** [comment]
Aight

**Comment by justchokingaround** [comment]
i do like this a lot
